api/_lib/parser.ts

```diff
@@ -35,6 +35,8 @@ export function parseRequest(req: IncomingMessage) {
         images: getArray(images),
         widths: getArray(widths),
         heights: getArray(heights),
+        imagesCircle: getArray(imagesCircle).map((x) => x === '1' || x === 'true'),
+        hideDots: hideDots === '1' || hideDots === 'true'
```

Comment:
I could see this being extended in the future, so lets use `bg=plain` and then we could have `bg=dots` be the default.

The reason is I could see that we might want something like `bg=transparent` or maybe other patterns in the future.

Reply:
@styfle will do!

Reply:
@styfle acc6b83